src/model/src/ASX_DATA_MOD.F

```diff
@@ -231,7 +231,8 @@ Module ASX_DATA_MOD
       Real, Parameter :: rg0        = 1000.0     ! [s/m]
       Real, Parameter :: rgwet0     = 25000.0    ! [s/m]
       Real, Parameter :: rsndiff    = 10.0       ! snow diffusivity fac
-      Real, Parameter :: rsnow0     = 10000.0    !  Changed from 1000 to 10000 - Helmig et al 2012
+!     Real, Parameter :: rsnow0     = 10000.0    !  Changed from 1000 to 10000 - Helmig et al 2012
```

Comment:
Just want to verify that this comment is beneficial to keep in. It seems like it is to me, but I tend to try to keep code as clean as possible.

Reply:
will keep the comment at the moment